### DISCUSSION TITLE
Accès à au moins 2 PDM (compteurs) ?

### DISCUSSION THREAD

**Discussion by tirso58** [bokub/conso-api#34] [Aide] [answered]
Accès à au moins 2 PDL (compteurs) ? Je n'ai pas trouvé dans l'intégration Home Assistant.

**Comment by bokub** [comment, marked as answer]
> Accès à au moins 2 PDL (compteurs) ?

Ah ouais, pas bonjour, pas de verbe, tu dois être vraiment très pressé pour faire si peu d'efforts et poser tes questions comme si j'étais au service client

Alors si, Conso API permet d'accéder à autant de compteurs que tu veux. Si ta question porte sur HA-Linky, tu es dans le mauvais repository et y'a déjà quelqu'un qui a posé la question y'a 2 jours ( https://github.com/bokub/ha-linky/issues/22 ), mais bon t'es pressé donc je comprends que tu n'aies pas le temps de chercher 30 secondes

Bonne journée à toi
